src/admin/localService/loadFile.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+const fs = require("fs");
+const pathJoin = require("path").join;
+
+const readdir = fs.promises.readdir;
+
+/**
+ * load file into server memory
```

Comment:
A better description would be that it takes a directory and recursively traverses it until it finds a cetain filename, and then returns the contents of that file.